bundles/org.openhab.binding.neohub/src/main/java/org/openhab/binding/neohub/internal/NeoHubBindingConstants.java

```diff
@@ -78,6 +85,25 @@
      */
     public static final String CHAN_TEMPERATURE_SENSOR = "sensorTemperature";
 
+    /*
+     * Heatmiser Device Types
+     */
+    public static final int HEATMISER_DEVICE_TYPE_CONTACT = 5;
+    public static final int HEATMISER_DEVICE_TYPE_PLUG = 6;
+    public static final int HEATMISER_DEVICE_TYPE_REPEATER = 10;
+    public static final int HEATMISER_DEVICE_TYPE_TEMPERATURE_SENSOR = 14;
+
+    /*
+     * configuration parameters
+     */
+    public static final String PARAM_HOLD_ONLINE_STATE = "holdOnlineState";
+
+    /*
+     * regular expressions
+     */
+    public static final String REGEX_HEATMISER_REPEATER = "^repeaternode\\d+";
+    public static final String REGEX_IP_ADDRESS = "\\b((25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)(\\.|$)){4}\\b";
```

Comment:
I would prefer if you changed this field to a compiled Pattern and used the compiled pattern for your regex operations.

Reply:
Ok.